### ISSUE TITLE
ESLint output is always about the first error found

### ISSUE THREAD

**Issue by fgblomqvist** [fi3ework/vite-plugin-checker#48]
Seems like this code incorrectly only looks at the `firstMessage` rather than the current one in the iteration (`d`):
https://github.com/fi3ework/vite-plugin-checker/blob/1403f6fe9bc66486e3dff5597f8927c28ff2dfe9/packages/vite-plugin-checker/src/logger.ts#L269-L310

**Comment by fi3ework** [comment]
you mean on the overlay or in terminal console or both? 🤔 overlay is not designed for show multiple errors so plugin show the first issue for now. But terminal should show all issue.

**Comment by fgblomqvist** [comment]
It's in the console. The overlay "correctly" shows just the first issue.

**Comment by fgblomqvist** [comment]
To clarify, the code above should look like this:
```ts
 return diagnostic.messages 
   .map((d) => { 
     let level = DiagnosticLevel.Error 
     switch (d.severity) { 
       case 0: // off, ignore 
         level = DiagnosticLevel.Error 
         return null 
       case 1: // warn 
         level = DiagnosticLevel.Warning 
         break 
       case 2: // error 
         level = DiagnosticLevel.Error 
         break 
     } 
  
     const loc: SourceLocation = { 
       start: { 
         line: d.line, 
         column: d.column, 
       }, 
       end: { 
         line: d.endLine || 0, 
         column: d.endColumn, 
       }, 
     } 
  
     const codeFrame = createFrame({ 
       source: diagnostic.source ?? '', 
       location: loc, 
     }) 
  
     return { 
       message: d.message, 
       conclusion: '', 
       codeFrame, 
       stripedCodeFrame: codeFrame && strip(codeFrame), 
       id: diagnostic.filePath, 
       checker: 'ESLint', 
       loc, 
       level, 
     } as any as NormalizedDiagnostic 
   }) 
```
The way it is right now, for each file, it will output the same message X times (where X is the number of lint errors/warnings in that file). I made the above modification myself and verified that it works correctly after that. I can submit a PR if you want.

**Comment by fi3ework** [comment]
closed in #49 and #52 by @fgblomqvist
